frontend/ui/widget/progresswidget.lua

```diff
@@ -92,4 +92,27 @@ function ProgressWidget:setPercentage(percentage)
     self.percentage = percentage
 end
 
+function ProgressWidget:updateStyle(thick, height)
+    if thick then
+        if height then
+            self.height = height
+        end
+        self.margin_h = Screen:scaleBySize(3)
```

Comment:
You call this `self.progress_bar:updateStyle(true, Screen:scaleBySize(bar_height))`.
As you already do many `Screen:scaleBySize` here, you might as well do that for the provided `height`, so it's a bit simpler at the calling places:
`self.progress_bar:updateStyle(true, bar_height)`.